gammapy/maps/tests/test_hpxnd.py

```diff
@@ -522,7 +521,14 @@ def test_hpxmap_read_healpy(tmp_path):
         format = HpxConv.identify_hpx_format(header)
         assert format == "healpy"
 
+    #default case: should take the first column "TEMPERATURE"
     m1 = Map.read(path)
     assert m1.data.shape[0] == npix
     diff = np.sum(m[0] - m1.data)
     assert_allclose(diff, 0.0)
+
+    #specifying the colname by default for healpy it is "Q_POLARISATION"
+    m2 = Map.read(path, colname="Q_POLARISATION")
+    assert m2.data.shape[0] == npix
+    diff = np.sum(m[1] - m2.data)
+    assert_allclose(diff, 0.0)
```

Comment:
Just to be sure: maybe fill the second map with ones or another number, so that the data is different from the first case?

Reply:
The second map is different as I subtract 1.

Reply:
Sorry, indeed I missed that...